src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -197,7 +197,9 @@ public sealed partial class ControlChars
     public sealed partial class Conversion
     {
         internal Conversion() { }
-        public static object CTypeDynamic(object? Expression, System.Type TargetType) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("The Expression origin object cannot be statically analyzed and may be trimmed.")]
```

Comment:
(nit) it isn't really the object that might be trimmed, but instead the members it points to. What do you think of:

`The Expression's underlying type cannot be statically analyzed and its members may be trimmed.`